mmv1/products/networksecurity/GatewaySecurityPolicy.yaml

```diff
@@ -105,5 +105,5 @@ properties:
     type: String
     description: |
       Name of a TlsInspectionPolicy resource that defines how TLS inspection is performed for any rule that enables it.
-    min_version: 'beta'
+      Note: TlsInspectionPolicy resource is still in [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html) therefore it will need to import the provider.
```

Comment:
Nit: could you change to `google_network_security_tls_inspection_policy` so that it's clear which resource we're talking about?

Reply:
Done